tests/org.eclipse.n4js.lang.tests/src/org/eclipse/n4js/tests/parser/templates/ParserTest.xtend

```diff
@@ -31,36 +31,36 @@ class ParserTest extends AbstractParserTest {
 	}
 
 	@Test
-	def void testNoSubstBackslash() {
+	def void testSubstBackslash() {
 		val script = '`\\\\`'.parseJSSuccessfully
 		val statement = script.scriptElements.head as ExpressionStatement
 		val template = statement.expression as TemplateLiteral
 		assertEquals(1, template.segments.size)
 		val segment = template.segments.head as TemplateSegment
-		assertEquals(segment.rawValue, 2, segment.rawValue.length)
-		assertEquals('\\\\', segment.rawValue)
+		assertEquals(segment.rawValue, 1, segment.rawValue.length)
```

Comment:
I had checked this and just double-checked it and the new expectation seems correct. Specifically, for the script
```
console.log(`\\`);
console.log(`\\` === "\\");
```
node will output:
```
\
true
```
So the new expectation is correct, or am I confusing something, here?

By the way: I noticed other incorrect expectations in this file. For example, the script
```
`${}`;
```
will produce a syntax error (instead of the literal evaluating to the empty string). I guess this is due to the fact that we implemented the template literals before the ES6 spec was fully finalized (and iirc they tweaked some details late in the standardization process).